node_modules/col-core/lib/api.js

```diff
@@ -355,6 +355,16 @@ var getStatus = module.exports.getStatus = function(callback) {
   CanvasAPI.getCanvases(function(err, canvases) {
     status.db = err ? false : true;
 
+    // Application version number
+    var versionsFile = path.join(__dirname, '../../..', 'versions/application.txt');
+    fs.readFile(versionsFile, "utf8", function(err, data) {
```

Comment:
@johncrossman Since this is asynchronous, I think the function might exit before the file is read. You'd either want to use `readFileSync` or put the rest of the `getStatus` code inside the callback.